doc/Changelog.md

```diff
@@ -3,6 +3,11 @@ Changelog {#changelog}
 
 # Release 1.5 (git master)
 
+* [266](https://github.com/BlueBrain/Tide/pull/266):
+  Multiple bugfixes:
+  - Fix crashes in case of file read errors on some of the wall processes.
+  - Fix InfiniBand networking using Intel MPI.
+  - Fix bug that could cause the background tap to become permanently disabled.
```

Comment:
The fix for the hardware swap barrier is not worth mentioning?

Reply:
there is no fix, it still doesn't work. I added a couple more checks but that's irrelevant.